source/Pickup.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "Entity.hpp"
+
+namespace GTA
+{
+	public ref class Pickup sealed : public Entity
```

Comment:
If all Entity related functions work on pickups too, then that can very well be. But only then =).